.github/workflows/NodeCI.yml

```diff
@@ -11,77 +11,85 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2
       - uses: actions/setup-node@v3
       - name: Install Packages
-        run: yarn install
+        run: pnpm install
       - name: Lint
-        run: yarn lint
+        run: pnpm run lint
       - name: Build
-        run: yarn build
+        run: pnpm run build
   test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x, 17.x, 18.x, 19.x]
+        node-version: [12.x 14.x, 16.x, 17.x, 18.x, 19.x]
```

Comment:
```suggestion
        node-version: [12.x, 14.x, 16.x, 17.x, 18.x, 19.x]
```

Reply:
I got this after adding a debug print😇
But still, CI is failed...